serving/docs/lmi/user_guides/vllm_user_guide.md

```diff
@@ -93,4 +99,9 @@ If you encounter an issue with a `Pass Through` configuration, it is likely an i
 | option.load_format	                      | \>= 0.26.0  | Pass Through	       | The checkpoint format of the model. Default is auto and means bin/safetensors will be used if found.	                                                                                                                                                                                                                                                                                                | Default: `auto`	       |
 | option.enforce_eager                     | \>= 0.27.0  | Pass Through	       | vLLM by default will run with CUDA graph optimization to reach to the best performance. However, in the situation of very less GPU memory, having CUDA graph enabled will cause OOM. So if you set this option to true, we will use PyTorch Eager mode and disable CUDA graph to save some GBs of memory.	                                                                                           | Default: `False`	      |
 | option.gpu_memory_utilization            | \>= 0.27.0  | Pass Through	       | This config controls the percentage of memory to be allocated to PagedAttention. Default to 0.9 (90%). We don't recommend to change this value because this impact the overall GPU memory allocations.                                                                                                                                                                                               | Default: `0.9`	        |
+| option.enable_loras                      | \>= 0.27.0  | Pass Through	       | This config enables support for LoRA adapters.                                                                                                                                                                                                                                                                                                                                                       | Default: `false`	      |
+| option.max_loras                         | \>= 0.27.0  | Pass Through	       | This config determines the maximum number of LoRA adapters to cache on GPU. All others will be evicted by LRU cache to memory or disk.                                                                                                                                                                                                                                                               | Default: `1`	          |
```

Comment:
if defaults is 1, eseentially customer are not really doing batch inference right? Since at runtime we need to load each of adapter back to GPU

Reply:
Yeah. 1 was their default, but I changed it to 4 which makes more sense for how we are using it.